.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
A yes/no checkbox or something to ask if the person is willing to work on it?

Reply:
I'd say that might be better to leave to comments rather than the description

Reply:
can discuss offline.